### ISSUE TITLE
Spawn Application function is not available

### ISSUE THREAD

**Issue by zangcc** [federicodotta/Brida#114]
Brida Version: 0.5
<img width="925" alt="image" src="https://github.com/federicodotta/Brida/assets/64825932/5140d418-9fd9-4c1f-bdc0-14ffacbd3c67">

Frida version:
 ./frida-server-16.1.2-android-arm64 

Brida's Attach application is fine, which means that my process PID is correct, the Server status and Application status are both green, and it can be successfully started, but the Spawn application cannot be used.

Click Spawn application to report an error: "Exception with spawn application net.razorvine.pyro.PyroException: [frida.ProcessNotFoundError] process not found."

It says the process is not found, why? I have confirmed that the process pid is correct.

**Comment by federicodotta** [comment]
Hi @zangcc,

To use the "spawn" application functionality you have to supply the application ID instead of the PID.

You have to supply the PID if you want to use the "attach" functionality, that attaches to an application already started. 

Federico

**Comment by zangcc** [comment]
I used the application ID, but the problem still exists. I hope you can help me solve this problem as soon as possible. Thank you very  @federicodotta 

<img width="1133" alt="image" src="https://github.com/federicodotta/Brida/assets/64825932/644df7af-f523-485c-8cfa-59e825dfb664">

**Comment by yayip** [comment]
Hi,

I have a same problem, Has anyone already solved this problem?

**Comment by zangcc** [comment]
Hello~ I have solved this problem. The reason for the problem is that there is a relationship between Frida and Android versions, not any Frida version is compatible with all Androids. Take me as an example, my Android version is Android 9, the Frida I use is Frida15.0.0, the Frida-compile version is 10.2.5, and the Frida-server version is 15.0.0-android-arm64. There is no problem at all when my configuration runs later, I hope it can help you, my friend.
I also wrote a tutorial blog about deploying brida and released a deployment video.
csdn: http://t.csdn.cn/nCHAt
bilibili: https://space.bilibili.com/392937810
